### ISSUE TITLE
Add title attribute to epub.js iframe

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/cozy-sun-bear#81]
`<iframe id="epubjs-view-f834724f-c9f5-4b2e-b4c6-e87254fe0b2e" scrolling="no" style="overflow: hidden; border: medium none; visibility: visible; width: 1668px; height: 270px;" srcdoc="<html xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:epub=&quot;http://www.idpf.org/2007/ops&quot;>`

- [ ] add the book title(?) as the value to the `title` attribute

Fix any of the following:
- [ ] aria-label attribute does not exist or is empty
- [ ] aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- [ ] Element has no title attribute or the title attribute is empty
- [ ] Element's default semantics were not overridden with role="presentation"
- [ ] Element's default semantics were not overridden with role="none"

**Comment by respinos** [comment]
Note: we can solve this inside cozy-sun-bear with hooks. Vaguely:

```
this._rendition.on("rendered", function(section, view) {
  var index = this._book.navigation.tocByHref[section.href];
  var label = this._book.toc[index].label;
  view.iframe.setAttribute("title", label);
}).bind(this);
```
